Client/src/Components/LoginCss.css

```diff
@@ -20,6 +20,12 @@
     font-size: 100px;
     font-weight: 700;
     letter-spacing: -7px;
+    background: url("../assets/bg1.webp") 50% 50%;
+    background-size: cover;
+    -webkit-text-fill-color: transparent;
```

Comment:
Ensure that the CSS properties for background styling include fallbacks or alternatives to handle browser compatibility issues. [important]